aggregator/programs/premerge.ts

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S deno run --unstable --allow-run --allow-read --allow-write
+#!/usr/bin/env -S deno run --unstable --allow-run --allow-read --allow-write --allow-env
```

Comment:
`--allow-env` now appears to be required for most scripts. Unsure what change did this, looks to be Deno std lib related.